extra-dev/packages/coq-certicoq/coq-certicoq.dev/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "The CertiCoq Team"
+homepage: "https://certicoq.org/"
+dev-repo: "git+https://github.com/CertiCoq/certicoq"
+bug-reports: "https://github.com/CertiCoq/certicoq/issues"
+authors: ["Andrew Appel"
+          "Yannick Forster"
+          "Anvay Grover"
+          "Joomy Korkut"
+          "John Li"
+          "Zoe Paraskevopoulou"
+          "Matthieu Sozeau"
+          "Matthew Weaver"
+          "Abhishek Anand"
+          "Greg Morrisett"
+          "Randy Pollack"
+          "Olivier Savary Belanger"
+  ]
+license: "MIT"
+build: [
+  ["bash" "./configure.sh"]
+  [make "all"]
+  [make "plugins"]
+  [make "bootstrap"]
+  [make "-C" "benchmarks" "all"] {with-test}
+  [make "-C" "bootstrap" "tests"] {with-test}
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "stdlib-shims"
+  "coq" {>= "8.17" & < "8.18~"}
+  "coq-compcert" {= "3.12"}
+  "coq-equations" {= "1.3+8.17"}
+  "coq-metacoq-erasure-plugin" {= "8.17.dev"}
+  "coq-metacoq-safechecker-plugin" {= "8.17.dev"}
+  "coq-ext-lib" {>= "0.11.8"}
+]
+
+synopsis: "A Verified Compiler for Gallina, Written in Gallina "
+url {
+  git: "https://github.com/CertiCoq/certicoq.git"
```

Comment:
```suggestion
  src: "git+https://github.com/CertiCoq/certicoq.git#master"
```
The standard way to specify this is to use `src`, and I recommend specifying a specific branch.